### PR TITLE
Environment_Engine: Control of duplicate control points before extruding to volume

### LINKED ISSUE #3185: Environment_Engine: Prevent ExtrudeToVolume extruding panels with no area
If a polyline has duplicate control points in error it will try to create a panel with an area of 0.

### PR DESCRIPTION
**Pull request by albinber**
### Issues addressed by this PR

Closes #3185

### CODE REVIEW

**Comment by albinber** [comment]
@bhombot check compliance

File: Environment_Engine/Compute/ExtrudeToVolume.cs [hunk 1]
```diff
@@ -36,12 +36,14 @@
 using System.ComponentModel;
 
 using BH.oM.Analytical.Elements;
+using ICSharpCode.Decompiler.CSharp.Syntax;
+using ICSharpCode.Decompiler.IL.ControlFlow;
```
Comment (FraserGreenroyd) [inline Environment_Engine/Compute/ExtrudeToVolume.cs:40]:
```suggestion
```

**Review by FraserGreenroyd — CHANGES_REQUESTED** [review]

**Comment by albinber** [comment]
@bhombot check compliance

**Comment by FraserGreenroyd** [comment]
@BHoMBot check core

**Comment by FraserGreenroyd** [comment]
@BHoMBot check required

**Comment by FraserGreenroyd** [comment]
@BHoMBot this is a DevOps instruction. I am requesting neutral checks on: versioning, installer, ready-to-merge, unit-tests

**Review by FraserGreenroyd — APPROVED** [review]
All looks good, great work @albinber and good discussion with @charliebrookerBH on working out the UX on this 😄

**Pull request merged**